tests/integration/test_interchain.py

```diff
@@ -174,6 +190,42 @@ def test_create_bitcoin_interchain_fails_with_bad_private_key(self):
             response,
         )
 
+    def test_create_binance_interchain_with_values(self):
+        url = "http://binance-node.dragonchain.com"
+        privkey = "c7603e7928cc9892982d5cda97cc50b9db81ec40ef3f7b9d515fb333151140bf"
+        response = self.client.create_binance_interchain(
+            name=BINANCE_INTERCHAIN_NAME_1, testnet=True, private_key=privkey, node_url=url, rpc_port=26657, api_port=11699
+        )
+        self.assertEqual(response, {"status": 201, "ok": True, "response": BINANCE_INTERCHAIN_BODY}, response)
+
+    def test_create_binance_interchain_with_defaults(self):
+        response = self.client.create_binance_interchain(name=BINANCE_INTERCHAIN_NAME_2, testnet=True)
```

Comment:
why are you providing the testnet var if you're trying to test default values?

Reply:
Oops yes -- fixed!